### PR TITLE
Code cleanup

### PR DESCRIPTION
**Pull request by aeternas**
Just some cleanup

### CODE REVIEW

File: Prox/Prox/Models/PlaceProviders.swift [hunk 1]
```diff
@@ -66,8 +66,10 @@ class SinglePlaceProvider: PlaceProvider {
             latLong = nil
         }
 
-        let photoURLStrings = (dict["images"] as? [[String:String]])?.flatMap { $0["src"] } ?? []
-        photoURLs = photoURLStrings.flatMap { URL(string: $0) }
+        let photos = dict["images"] as? [[String:String]]
+
+        photoURLs = photos.flatMap { $0["src"] } ?? []
+            .flatMap { URL(string: $0) }
```
Comment (mcomella) [inline Prox/Prox/Models/PlaceProviders.swift]: This doesn't compile:

> Cannot subscript a value of type '[[String : String]]' with an index of type 'String'

File: Prox/Prox/Models/PlaceProviders.swift [hunk 2]
```diff
@@ -114,7 +116,11 @@ class CompositePlaceProvider: PlaceProvider {
     private(set) var totalReviewCount: Int = 0
 
     init(fromProviders providers: [PlaceProvider]) {
-        for provider in providers {
+        pullDataForProviders(providers)
+    }
+
+    fileprivate func pullDataForProviders(_ providers: [PlaceProvider]) {
+        providers.forEach { (provider) in
```
Comment (mcomella) [inline Prox/Prox/Models/PlaceProviders.swift]: nit: no parens around `provider`

File: Prox/Prox/Models/PlaceProviders.swift (hunk 1 above)
Comment (aeternas) [inline Prox/Prox/Models/PlaceProviders.swift]: Fixed, thx

File: Prox/Prox/Models/PlaceProviders.swift (hunk 2 above)
Comment (aeternas) [inline Prox/Prox/Models/PlaceProviders.swift]: Fixed

**Comment by mcomella** [comment]
Merged with d1c0bb9f8a15cc8893a892c3ca218c3995de47ff.

**Pull request closed without merging**